src/server/bin.js

```diff
@@ -2,27 +2,53 @@
 
 'use strict'
 
-// Usage: $0 [<address> <address 2>...]
+// Usage: $0 [-libp2pPort <port>] [-metricsPort <port>] [--disableMetrics]
```

Comment:
Isn't it better to specifiy a multiaddr instead of a port? So we save having a host option